videos-list/2023/35-dante_bayes.md

```diff
@@ -0,0 +1,38 @@
+Speaker: Dante Gates
+
+Title: The Power of Bayes in Industry: Your Business Model is Your Data Generating Process
+
+Video: https://youtu.be/7KrspD1TZNU
+
+## Event description:
+This talk will attempt to answer the question “what is a Data Generating Process and why does it matter?” While we will begin our discussion with a bit of theory, don’t worry about this being too technical or inaccessible if you’re new to Bayesian Statistics. Our primary goal is to focus on the second half of the question and give you tools to use for real-world applications.
+
+With the core concepts and background covered, we’ll demonstrate how incorporating this understanding into our modeling decisions allows us to embed elements of a business function directly into our statistical models and how this can provide immense value in industry settings, especially where traditional machine learning techniques fail, such as
+
+The ability to tackle critical problems when data is lacking, like launching a new product
+
+Building powerful, predictive models that are difficult to overfit
+
+Explainability is built in, and it’s already expressed in the terms of your business
+
+Best of all is that the design techniques we propose here are such that when you get one the benefits above, the rest usually come for free.
+
+## Discourse Discussion
+https://discourse.pymc.io/t/pymcon-web-series-the-power-of-bayes-in-industry-your-business-model-is-your-data-generating-process-feb-9-2023/11264
+
+## Timestamps
+0:00 
+x:xx 
+x:xx Help us add timestamps here: https://github.com/pymc-devs/video-timestamps
```

Comment:
Should be:
## Timestamps
```
0:00 
x:xx 
x:xx 
```
Help us add timestamps here: https://github.com/pymc-devs/video-timestamps